x-pack/legacy/plugins/uptime/public/pages/404.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiEmptyPrompt,
+  EuiPanel,
+  EuiTitle,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButton,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export const NotFoundPage = () => (
```

Comment:
Do you think we should name the file the same as the component it exports, i.e. `not_found.tsx`?

Reply:
makes sense